### PR TITLE
added comments

### PR DESCRIPTION
**Pull request by jonnyfd**
Does it look like I understand how this works?

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
 import select
 
-
+# the number of connections the server will wait for before starting game
 num_players = 2
+
+# list of clients
 sock_list = list()
+
+# socket throat clearing jargon, sets server to listen for connections on port 6699 via TCP and opens a "stream" of bytes
```
Comment (krglaws) [inline server.py:11]: the AF_INET parameter means IPv4, AF_INET6 would mean IPv6. SOCK_STREAM means it will accept TCP connections (as opposed to UDP).

Comment (jonnyfd) [reply server.py:11]: ah. i see

**Comment by krglaws** [comment]
yes, looks like you get it

File: server.py (hunk 1 above)
Comment (jonnyfd) [reply server.py]: check new commit

File: server.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # list of clients
 sock_list = list()
 
-# socket throat clearing jargon, sets server to listen for connections on port 6699 via TCP and opens a "stream" of bytes
+# specifies IPv4 over TCP
```
Comment (krglaws) [inline server.py:11]: perf

**Pull request merged**